src/lib.rs

```diff
@@ -13,9 +13,11 @@ pub mod prefs;
 /// Utilities to access resource files
 pub mod resources;
 /// Utilities to write tests.
-pub mod test;
-/// Web view types to handle web browsing contexts and window.
+// pub mod test;
```

Comment:
do we not use this anymore?

Reply:
We will refactor test module later on. Current one is incompatible with multiview.